services/galley/src/Galley/API/Public.hs

```diff
@@ -92,7 +92,32 @@ servantSitemap =
         GalleyAPI.getTeamConversation = Teams.getTeamConversation,
         GalleyAPI.deleteTeamConversation = Teams.deleteTeamConversation,
         GalleyAPI.postOtrMessageUnqualified = Update.postOtrMessageUnqualified,
-        GalleyAPI.postProteusMessage = Update.postProteusMessage
+        GalleyAPI.postProteusMessage = Update.postProteusMessage,
+        GalleyAPI.teamFeatureStatusSSOGet =
```

Comment:
Why is there no "PUT" counterpart for the sso feature?  The old code looks to me like it has that, but I vaguely remember that we did only allow it in the internal API.  (Same for a few more features below.)

Reply:
One intermediate step in rewriting these endpoints to Servant was to get rid of public endpoints that are not meant for the public, and these were always setters. By inspecting the code, I and whoever paired with me at that point concluded that SSO should not have a public endpoint, and similarly for some other features. Note that they are still available as internal endpoints at `PUT /i/teams/:tid/feature...`.
If this is not a good conclusion, I will be happy to reintroduce removed public endpoints for any/all team features.

Reply:
@fisx and @pcapriotti , for the context of why certain PUT endpoints were removed, see https://wearezeta.atlassian.net/browse/SQSERVICES-463. I hope that clarifies it!

Reply:
@fisx , I will mark this as resolved per SQSERVICES-463, but if you feel like this is not resolved, please let me know!

Reply:
That means https://wearezeta.atlassian.net/browse/SQSERVICES-463 can be closed with this PR, right?

Nice side-effect of the refactoring!  :)

Reply:
> That means https://wearezeta.atlassian.net/browse/SQSERVICES-463 can be closed with this PR, right?
> 
> Nice side-effect of the refactoring! :)

Yes, that is correct.